terraform/projects/infra-datagovuk-organogram-bucket/main.tf

```
@@ -63,9 +63,12 @@ data "terraform_remote_state" "infra_monitoring" {
 resource "aws_s3_bucket" "datagovuk-organogram" {
   bucket = "datagovuk-${var.aws_environment}-ckan-organogram"
 
-  cors_rule {
+  dynamic "cors_rule" {
+    for_each = [for s in ["${compact(list(var.domain, var.aws_environment == "production" ? "https://staging.data.gov.uk" : ""))}", "https://find.eks.${var.aws_environment}.govuk.digita"] : {
```

You've got some obsolete `"${}"` syntax there and what looks like a typo in the domain name.
```suggestion
    for_each = [for s in [compact(list(var.domain, var.aws_environment == "production" ? "https://staging.data.gov.uk" : "")), "https://find.eks.${var.aws_environment}.govuk.digital"] : {
```

---

ok, I'll fix it